tuned/utils/profile_recommender.py

```diff
@@ -32,7 +32,13 @@ def recommend(self, hardcoded = False):
 		profile = consts.DEFAULT_PROFILE
 		if hardcoded:
 			return profile
-		matching = self.process_config(consts.RECOMMEND_CONF_FILE)
+		has_root = os.getuid() == 0
+		if not has_root:
+			log.warning(("Profile recommender is running without root"
+						 " privileges. Profiles with virt recommendation"
+						 " condition will be omitted."))
```

Comment:
I prefer to put log messages on a single line, even if it makes the line too long. It makes it easier to `grep` for the message.

Reply:
Fixed